task.go

```diff
@@ -62,6 +64,11 @@ func (e *Executor) Run(ctx context.Context, calls ...taskfile.Call) error {
 		}
 	}
 
+	if e.Summary {
+		summary.PrintAll(e.Logger, e.Taskfile, calls)
```

Comment:
I've been thinking hard to do this.
I think the task component (high level abstraction, only delegates to other components) should not care about a low level detail like spacing for the output of the summary.
To fix this, I added a `PrintAll` Method for the `summary`-package, which deals with it.